x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/editor_frame.scss

```diff
@@ -0,0 +1,74 @@
+.lnsPage {
+  padding: 0;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  overflow: hidden;
+}
+
+.lnsSidebar {
+  margin: 0;
+  flex: 1 0 18%;
+  min-width: ($euiSize * 16);
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+}
```

Comment:
Since you're adding `overflow: hidden` here, please also add a scrollbar to the data panel- without a scrollbar it's cutting off labels both horizontally and vertically. 
<img width="299" alt="Screenshot 2019-06-25 12 42 25" src="https://user-images.githubusercontent.com/666475/60116769-b283d580-9746-11e9-8f10-3eaae064c375.png">